java/athena/src/main/java/sleeper/athena/record/FieldAsString.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2022-2025 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sleeper.athena.record;
+
+/**
+ * Storage of key pairing for field and value.
```

Comment:
I don't know what "key pairing" means here. It's a value for a field, held as a string along with the name of the field.